### DISCUSSION TITLE
how open 444G 7z file,There are a lot of files

### DISCUSSION THREAD

**Discussion by hktalent** [klauspost/compress#710] [Q&A] [answered]
how open 444G 7z file,There are a lot of files

![](https://user-images.githubusercontent.com/18223385/206704881-5c5dc276-857f-4437-a70d-bb2a10778f78.png)

thanks

**Comment by klauspost** [comment, marked as answer]
There is no 7zip support in this package.

A google search turned up: https://pkg.go.dev/github.com/bodgit/sevenzip#section-readme

**Reply by hktalent** [reply]
thanks
